README.md

```
@@ -1 +1 @@
-Welcome to StrideHabits!
+Welcome to StrideHabits! 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)